example.docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+service:
```

Comment:
```services```
Otherwise invalid file and 'Additional property service is not allowed' error appears.